src/index.js

```diff
@@ -154,7 +154,7 @@ export function configure(env) {
 
 			alias: {
 				preact: path.resolve(__dirname, 'preact.js'),
-				react: 'preact',
+				react: 'preact-compat',
```

Comment:
Do we need to/should we do the same thing with `preact-compat` that we did with `preact`: pass it down in `context`, declare as a global, and do something like:
```
alias: {
    preact: path.resolve(__dirname, 'preact.js'),
    react: path.resolve(__dirname, 'preact-compat.js'),
```

Reply:
I don't know that it is required in the same way that it is required for preact because we don't want more than one instance of it on the page, but maybe we should do it to save the overhead of compiling it into every zimlet

Reply:
Yes I was thinking the same, we can take advantage of the same mechanism to inject `preact-compat`.

I would like to take it a step further and alias `@zimbra/blocks` to a global containing all of `@zimbra/blocks`, which would let consumers do a regular import of `@zimbra/blocks` inside a Zimlet.